app/helpers/rails_admin/application_helper.rb

```diff
@@ -35,7 +35,12 @@ def edit_user_link
       return nil unless _current_user.respond_to?(:email)
       return nil unless abstract_model = RailsAdmin.config(_current_user.class).abstract_model
       return nil unless (edit_action = RailsAdmin::Config::Actions.find(:edit, controller: controller, abstract_model: abstract_model, object: _current_user)).try(:authorized?)
-      link_to _current_user.email, url_for(action: edit_action.action_name, model_name: abstract_model.to_param, id: _current_user.id, controller: 'rails_admin/main')
+      link_to url_for(action: edit_action.action_name, model_name: abstract_model.to_param, id: _current_user.id, controller: 'rails_admin/main') do
+        html = []
+        html << image_tag("#{(request.ssl? ? 'https://secure' : 'http://www')}.gravatar.com/avatar/#{Digest::MD5.hexdigest _current_user.email}?s=30", alt: '') if _current_user.email.present?
```

Comment:
It could be nice to go for a more modern/consolidated user drop down. I'll try and find an example, but you've probably seen them before - the ones where you click the username/image and a drop down appears with a profile and logout link. It would solve the red "danger-label" as mentioned above and tidy up the top nav bar... Just a thought!